it/uk/gov/hmrc/agentsubscription/controllers/SubscriptionControllerForOverseasISpec.scala

```diff
@@ -484,6 +554,7 @@ class SubscriptionControllerForOverseasISpec extends BaseISpec with OverseasDesS
     verifyDeleteKnownFactsCalled(deleteKnownFact)
     verifyCreateKnownFactsCalled(createKnownFact)
     verifyEnrolmentCalled(enrol)
+    verify(amls, postRequestedFor(urlEqualTo(s"/agent-assurance/overseas-agents/amls")))
```

Comment:
Could have a verify method in the AgentAssuranceStub trait, so the URL need not be duplicated.